gax/test/showcase-echo-client/package.json

```diff
@@ -28,16 +28,16 @@
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r protos build/ && minifyProtoJson",
     "compile-protos": "compileProtos src",
-    "prefetch": "rm -rf node_modules package-lock.json google-gax*.tgz gapic-tools*.tgz && cd ../.. && npm pack && mv google-gax*.tgz test/showcase-echo-client/google-gax.tgz && cd ../tools && npm install && npm pack && mv gapic-tools*.tgz ../gax/test/showcase-echo-client/gapic-tools.tgz",
-    "prepare": "npm run compile-protos && npm run compile"
+    "prepare": "npm run compile-protos && npm run compile",
```

Comment:
Any reason to swap these two? I think they were arranged in an alphabetical order, `prefetch` before `prepare`.

Reply:
no good reason, I'll swap them back!